src/lib/hooks/swap/useBestTrade.ts

```diff
@@ -81,9 +81,9 @@ export function useBestTrade(
     (debouncedOtherCurrency && otherCurrency && debouncedOtherCurrency !== otherCurrency)
 
   const syncing = isTradeDebouncing({
-    amounts: [amountFromLatestTrade, debouncedAmount],
-    indepdenentCurrencies: [currencyFromTrade, debouncedOtherCurrency],
-    dependentCurrencies: [otherCurrencyFromTrade, otherCurrency],
+    amounts: [amountFromLatestTrade, amountSpecified],
```

Comment:
What's going on here? Was this a mistake from a past PR? Seems out of place.

Reply:
Yeah this is a mistake - this logic updates the logic to match the current app which does make the loading work correctly. 

Without this we enter a perpetual loading state